web/src/components/model/model.models.ts

```diff
@@ -0,0 +1,20 @@
+import { Basement, Category, Model } from '../../api/typings';
+import { Job } from '../../api/typings/jobs';
+
+export type ModelValues = {
+    baseModel?: Model;
+    name: string | undefined;
```

Comment:
you can define optional(or maybe undefined) type like this `key?: value`.

`?:` means that this key is optional, which means it can be undefined